### DISCUSSION TITLE
Context in @validate_call

### DISCUSSION THREAD

**Discussion by TylerGrantSmith** [pydantic/pydantic#7180] [Feature Request]
Is it possible or will it be possible to supply context  in methods decorated with `@validate_call`?

**Comment by harunyasar** [comment]
> Is it possible or will it be possible to supply context in methods decorated with `@validate_call`?

What do you mean by `context`?

**Reply by TylerGrantSmith** [reply]
Validation context?

https://docs.pydantic.dev/latest/usage/validators/#validation-context
